packages/demo/src/components/examples/SeparatorExamples.tsx

```diff
@@ -4,7 +4,6 @@ import {Badge, Separator} from 'react-vapor';
 export const SeparatorExamples = () => (
     <div className="mt2">
         <div className="form-group">
```

Comment:
Would it be relevant to switch this part to `Section` also?
```suggestion
        <Section>
             <Section>   // maybe only one will suffice
                   <Badge
```

Reply:
I did not changed it because it was already fine as is but I can do it if it's better with Section :+1: